### PR TITLE
new create_order method

### PR DESCRIPTION
**Pull request by arturictus**

### CODE REVIEW

File: mix.exs [hunk 1]
```diff
@@ -4,7 +4,7 @@ defmodule ExBinance.MixProject do
   def project do
     [
       app: :ex_binance,
-      version: "0.0.5",
```
Comment (rupurt) [inline mix.exs:7]: :scissors: I like to do this in a separate PR

File: lib/ex_binance/timestamp.ex [hunk 2]
```diff
@@ -1,3 +1,3 @@
 defmodule ExBinance.Timestamp do
-  def now, do: :os.system_time(:millisecond)
+  def now(), do: System.os_time(:millisecond)
```
Comment (rupurt) [inline lib/ex_binance/timestamp.ex:2]: :+1:

File: lib/ex_binance/private.ex [hunk 3]
```diff
@@ -21,6 +21,11 @@ defmodule ExBinance.Private do
                 credentials
               ),
               to: ExBinance.Rest.CreateOrder
+  defdelegate create_order(
```
Comment (rupurt) [inline lib/ex_binance/private.ex:24]: Can you add a test for this new function?

File: lib/ex_binance/rest/create_order.ex [hunk 4]
```diff
@@ -22,6 +22,16 @@ defmodule ExBinance.Rest.CreateOrder do
     |> parse_response()
```
Comment (rupurt) [inline lib/ex_binance/rest/create_order.ex:22]: How do you feel about calling the new function here so we share code paths?

**Review by rupurt — CHANGES_REQUESTED** [review]

**Comment by rupurt** [comment]
@arturictus can you also rebase this PR? I've fixed up the test runner on CI

**Comment by arturictus** [comment]
> @arturictus can you also rebase this PR? I've fixed up the test runner on CI

Hi @rupurt, I did not want to submit the PR yet I'm still working on it and I will have to change few things to make it work with my project, those will be breaking changes.

I let you know when I have it ready and we can decide if the changes are suited for merge.

I'll try the new CI runner :)

**Comment by rupurt** [comment]
No worries @arturictus. Thanks for the update.

**Comment by rupurt** [comment]
Closing in favor of #43. Thanks for the initial idea in this PR @arturictus

**Pull request closed without merging**